timescaledb/tutorials/grafana/visualizations/time-series.md

```diff
@@ -0,0 +1,244 @@
+# Build a time-series graph in Grafana 
+A time-series graph is a line graph that plots points changing over time. It
+allows you to see trends and fluctuations in your data. It's usually plotted in
+2 dimensions. The X axis represents time, and the Y axis represents the
+value of your data.
```

Comment:
```suggestion
allows you to see trends and fluctuations in your data. It's usually plotted in
two dimensions. The X-axis represents time, and the Y-axis represents the
value of your data.
```

See below for my comments on "2" vs. "two". 

For the axis, I honestly don't know the official rule. But in nearly everything I read/see, folks use the hyphen. @charislam - any input here?

Reply:
Did some style guide spelunking. Chicago Manual of Style, which is our backup style guide, recommends _x_-axis and _y_-axis (note the lowercase). They are happy to go either way with the hyphen as long as it's consistent. I think it looks most conventional in lowercase with the hyphen, so I would recommend that.